pkg/cloud/packet/client.go

```diff
@@ -278,23 +294,26 @@ func (p *Client) EnableProjectBGP(projectID string) error {
 	}
 
 	// we did not have a valid one, so create it
-	req := packngo.CreateBGPConfigRequest{
-		Asn:            outLocalASN,
-		Md5:            outBGPPass,
+	useCase := "kubernetes-load-balancer"
+	apiRequest := p.BGPApi.RequestBgpConfig(ctx, projectID)
+	apiRequest.BgpConfigRequestInput(metal.BgpConfigRequestInput{
+		Asn:            int32(outLocalASN),
+		Md5:            &outBGPPass,
 		DeploymentType: "local",
-		UseCase:        "kubernetes-load-balancer",
-	}
-	_, err = p.BGPConfig.Create(projectID, req)
+		UseCase:        &useCase,
+	})
+	_, err = apiRequest.Execute()
```

Comment:
The goal of this PR is a 1:1 conversion, but looking at this code I am reminded of this https://github.com/equinix/terraform-provider-equinix/issues/121#issuecomment-1315818528 (use the BGP Discover API when affecting BGP project properties). This may be more applicable when a node is added and BGP Sessions is enabled for that node.  I'm only mentioning this for awareness.